plugins/outputs/whatap/whatap.go

```diff
@@ -0,0 +1,168 @@
+package whatap
+
+import (
+	"fmt"
+	"net"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/outputs"
+
+	whatap_io "github.com/whatap/go-api/common/io"
+	whatap_pack "github.com/whatap/go-api/common/lang/pack"
+	whatap_hash "github.com/whatap/go-api/common/util/hash"
+)
+
+const (
+	NetSrcAgentOneway = 10
+)
+
+type Whatap struct {
+	License string          `toml:"license"`
+	Servers []string        `toml:"servers"`
+	Pcode   int64           `toml:"pcode"`
+	Timeout config.Duration `toml:"timeout"`
+	Log     telegraf.Logger `toml:"-"`
+	oname   string
+	oid     int32
+	conn    net.Conn
+	dest    int
+}
+
+var sampleConfig = `
```

Comment:
Please make this a constant.
```suggestion
const sampleConfig = `
```